File: date-picker/src/main/java/me/foolishchow/android/datepicker/data/DateWheelVo.java
package me.foolishchow.android.datepicker.data;

import androidx.annotation.NonNull;

import com.contrarywind.interfaces.IPickerViewData;

public class DateWheelVo implements IPickerViewData {
    @NonNull
    public final String label;
    public final int value;

    public DateWheelVo(String label, int value) {
        this.label = label;
        this.value = value;
    }

    @Override
    public String getPickerViewText() {
        return label;
    }

    @Override
    public String toString() {
        return "DateWheelVo{label='" + label + '\'' + ", value=" + value + '}';
    }
}

File: date-picker/src/main/java/me/foolishchow/android/datepicker/data/DateWheelVo.kt
package me.foolishchow.android.datepicker.data

import com.contrarywind.interfaces.IPickerViewData

open class DateWheelVo(
        @JvmField
        open val label: String,
        @JvmField
        open val value: Int
) : IPickerViewData {
    override fun getPickerViewText(): String {
        return label
    }

    override fun toString(): String {
        return "DateWheelVo{label='$label', value=$value}"
    }
}